### ISSUE TITLE
Better support for keyboard navigation in the search pane

### ISSUE THREAD

**Issue by protist** [gentakojima/emojione-picker-ubuntu#21]
Hi, firstly, great program! It really fills a niche for a Linux emoji picker/keyboard! I love it!

However, I've just had some issues with keyboard navigation of the search pane. Firstly, when you mouse over the emoji, you get a tooltip with the name of the emoji. It'd be great to have something similar when navigating by keyboard, as the visual difference between similar emoji is often quite subtle. I think the current tooltip is probably sub-optimal for keyboard navigation, because they would obscure the emoji and restrict clear navigation. Perhaps a subpane below the main window?

Secondly, there are some issues with highlighting when navigating by keyboard. For example, I type in a string, then have to press down twice to select an emoji (is that a bug too?). However, if I press up a few times again, it keeps the last emoji selected, but also the text. Similarly, if I press down again, both the text field and emoji are selected.

![Screenshot](http://i.imgur.com/jnK3GS4.png]Screenshot)

In this screenshot, relating to my first point, it's hard to discern the difference between face 3 and 4 in the second last row.

**Comment by gentakojima** [comment]
Thanks for your testing! 

As for you second issue. Are you using the latest version in master? Keyboard navigation has always been awful and I made a few changes a month ago or so, but it still might be a little messy. I'll test it again as soon as possible.

For the first one, I think it's a good idea to add a lower bar with a text description. I'll also look into it.

**Comment by protist** [comment]
Yes, I'm using the latest version. Oh, I should mention, I [created](https://aur.archlinux.org/packages/emojione-picker-git/) an Arch Linux package for this project too, in the AUR. I just packaged the latest vcs version, since there hadn't been a version tag for a little while, but I'm happy to make one for the tagged version too if you like. Feel free to add installation information for Arch Linux to the Readme if you want, but Arch users will probably just check the AUR anyway.

Thanks for the reply; I'm looking forward to the changes!
